modules/ingestion/pipeline/variables.tf

```diff
@@ -29,6 +29,12 @@ variable "enable_logging" {
   default     = true
 }
 
+variable "cloudwatch_kms_key_id" {
+  description = "(Optional) Use a custom KMS key ID to encrypt Cloudwatch logs. If not specified, then it defaults to using the AWS KMS key."
```

Comment:
this description doesn't seem correct. It seems that it's not optional, it must be provided now.